.prettierignore

```diff
@@ -8,4 +8,4 @@ pnpm-lock.yaml
 # end managed by skuba
 
 /integration/base/
-
+/packages/**/*/lib*
```

Comment:
Is it worth putting this in `gitignore`? The `prettierignore` should then inherit it.